ocaml/xapi/storage_migrate.ml

```diff
@@ -539,17 +539,35 @@ let copy' ~task ~dbg ~sr ~vdi ~url ~dest ~dest_vdi ~verify_dest =
     finally
       (fun () ->
         debug "activating RW datapath %s on remote" remote_dp ;
-        ignore (Remote.VDI.attach2 dbg remote_dp dest dest_vdi true) ;
-        Remote.VDI.activate dbg remote_dp dest dest_vdi ;
-        with_activated_disk ~dbg ~sr ~vdi:base_vdi ~dp:base_dp (fun base_path ->
-            with_activated_disk ~dbg ~sr ~vdi:(Some vdi) ~dp:leaf_dp (fun src ->
```

Comment:
Looks like this is removed in this ("Add support for specifying nbd export in sparse_dd") patch, but added back in the next one?

Reply:
Yes, I need to clean up the commits.